src/SmartComponents/HybridInventoryTabs/helpers.test.js

```diff
@@ -144,3 +144,34 @@ describe('useActionResolver', () => {
     expect(handleModalToggle).toHaveBeenCalledWith(true, 'test-device-id');
   });
 });
+
+const defaultColumns = [
+  { key: 'updated', renderFunc: jest.fn() },
+  { key: 'system_profile', props: { width: 15 } },
+  { key: 'groups', props: { width: 15 } },
+];
+
+describe('mergeAppColumns', () => {
+  test('Should use last seen column render function for impacted column', () => {
+    const result = mergeAppColumns(defaultColumns);
+
+    const impacted_date = result.find(
+      (column) => column.key === 'impacted_date'
+    );
+    expect(impacted_date.renderFunc).toEqual(defaultColumns[0].renderFunc);
+  });
+  test('Should extend OS column props to disable sorting', () => {
+    const result = mergeAppColumns(defaultColumns);
+
+    const system_profile = result.find(
+      (column) => column.key === 'system_profile'
+    );
+    expect(system_profile.props).toEqual({ width: 15, isStatic: true });
+  });
+  test('Should extend groups column props to disable sorting', () => {
+    const result = mergeAppColumns(defaultColumns);
+
+    const groups = result.find((column) => column.key === 'groups');
+    expect(groups.props).toEqual({ width: 15, isStatic: true });
+  });
+});
```

Comment:
Would be great if we could split the hook and tests related work into a separate commit. It confused me a bit why it's here whereas the PR only states that you fix the columns width

Reply:
Hmm, I am not sure if this gives us any benefit? The PR is not large and furthermore once approved I will squash the PR into the master.

Reply:
OK, it was a nit-pick rather, feel free to move on